Chapter08/recipe-06/cxx-example/src/conversion/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+target_sources(
+  automaton
+  PRIVATE
+    ${CMAKE_CURRENT_LIST_DIR}/conversion.cpp
+    ${CMAKE_CURRENT_LIST_DIR}/conversion.hpp
```

Comment:
Is the header only used here? Or is it used in other sources outside this directory?

Reply:
Also used outside this directory.